shell/platform/windows/win32_window.cc

```diff
@@ -83,13 +79,17 @@ LRESULT CALLBACK Win32Window::WndProc(HWND const window,
 
     auto that = static_cast<Win32Window*>(cs->lpCreateParams);
 
-    // Since the application is running in Per-monitor V2 mode, turn on
-    // automatic titlebar scaling
-    BOOL result = that->dpi_helper_->EnableNonClientDpiScaling(window);
-    if (result != TRUE) {
-      OutputDebugString(L"Failed to enable non-client area autoscaling");
+    if (that->dpi_helper_->IsPerMonitorV2Available()) {
```

Comment:
Assuming you move the DPI mode setting logic into the client app, this would become a getter for the current DPI mode